tutorials/receiving-money.md

```diff
@@ -0,0 +1,275 @@
+    amount: 10000,
+    metadata: {
+        custom_filters: {
+            recurring: true,	//only accept cards that support recurring debit
+            banks: ['057', '100'],  // bank codes to accept in Pay with Bank Channel
+         	card_brands: ["mastercard", "visa"] //only accept mastercard and visa cards
+        }
+    }
+}
+```
+
+ 
+
+### Response Structure
+
+A typical Paystack response object (response) contains:
+
+- status
+- message
+- data
+
+**Status** is true or false depending on if the API call is was processed or not. If you something was wrong with your request (no authorization object, no/wrong secret key, invalid/duplicate transaction reference) the status will be false. The status is NOT the status of the transaction, it is the status of the API call.
+
+If the response status is true, the response object will contain a **data** field. This contains the response for the request you made. The data object has a status field `response->data->status` that contains the status of the transaction.
+
+For example, if a payment is successful, the response status will be true, and the response data status will be success. If the payment fails, the response status will be true and the response data status will be false. The data also contains the rest of the response for the request you made.
+
+### Common Errors
+
+- The most common mistake people make is not including the Authorization Header. All API requests on Paystack are authenticated with your secret key. Format is `Authorization: "Bearer YOUR_SECRET_KEY"`  Don’t leave out the `Bearer` keyword
+- `amount` value is in kobo. Always remember to multiply the Naira value of the amount by 100. So if you want to charge N100, you will pass amount as `100*100 = 10000`.
+- Calls to the Paystack API **should not be made from your frontend**, only from your server. This is because your secret key should never be exposed publicly. It can be used to access sensitive information on your Paystack account. The public key can only be used to initialize transactions on the Javascript Paystack Inline payment method.
+- Ensure to call the verify function from your callback function (for frontend implementation) or your callback URL/route (for standard redirect implementation). Only when control has been passed to your callback can you be certain that the user has completed payment. If you try to do a long pool to the verify endpoint, you will keep getting abandoned status until the user has completed the payment.
+
+### Use Case
+
+- An online store
+
+## Payment Channels
+
+Paystack provides various payment channels to allow merchant receive payment from customers.
+
+Currently we have
+
+- Pay with Card
+- Pay with Bank - this allows users complete payments using their Nigerian bank accounts
+- Pay with USSD
+- Pay with Ghana Mobile Money.
+
+The purpose of multiple channels is to allow the users complete payments using any of the channels in case any of them is down.
```

Comment:
The purpose of multiple channels is to allow the users complete payments using any of the channels in case any of them is **_unavailable_**.


**_suggestion_**